config/default/crds/serving_v1alpha1_kfservice.yaml

```diff
@@ -6,10 +6,25 @@ metadata:
     controller-tools.k8s.io: "1.0"
   name: kfservices.serving.kubeflow.org
 spec:
+  additionalPrinterColumns:
+  - JSONPath: .status.uri.external
+    name: uri
+    type: string
+  - JSONPath: .status.conditions[?(@.type==\"READY\")].status
+    name: status
+    type: string
+  - JSONPath: .status.default.traffic
+    name: defaulttraffic
+    type: integer
```

Comment:
I assume this is going be % of traffic... and we won`t allow decimals (e.g. 99.5%)?

Reply:
no decimals, the split percent is defined as integer :).